apps/src/StudioApp.js

```diff
@@ -2684,3 +2685,25 @@ StudioApp.prototype.polishGeneratedCodeString = function (code) {
     return code;
   }
 };
+
+/**
+ * Sets a bunch of common page constants used by all of our apps in our redux
+ * store based on our app options config.
+ * @param {AppOptionsConfig}
+ */
+StudioApp.prototype.setCommonPageConstants = function (config) {
```

Comment:
Another approach we could take we be to give this method the following signature:
```
StudioApp.prototype.setPageConstants = function (config, appSpecificConstants)
```
such that we only call setPageConstants from studioApp (with the app passing any non-common constants). Thoughts on their relative merits?